make/langtools/tools/javacserver/comp/SjavacImpl.java

```diff
@@ -23,57 +23,54 @@
  * questions.
  */
 
-package com.sun.tools.sjavac.comp;
+package javacserver.comp;
 
-import com.sun.tools.sjavac.Log;
-
-import java.io.*;
-import javax.tools.JavaFileObject;
+import com.sun.tools.javac.Main;
+import javacserver.Util;
+import javacserver.Log;
+import javacserver.Result;
+import javacserver.options.Option;
+import javacserver.server.Sjavac;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.stream.Stream;
```

Comment:
This is a build tool, so you get to set the guidelines, but generally, this is an unusual order for imports

Reply:
Haha! Calling this "unusual" is the most diplomatic expression I've heard today. :-) This is just bonkers. I used a combination of "organize imports" on IntelliJ (without setting up specific rules for the project) and a brute search-and-replace. I'll fix.